src/sdk/modules/smartSessionsValidator/toSmartSessionsValidator.ts

```diff
@@ -123,14 +123,14 @@ export const toSmartSessionsValidator = (
     getStubSignature: async () =>
       encodeSmartSessionSignature({
         mode,
-        permissionId,
+        permissionId: permissionIds[permissionIdIndex],
         enableSessionData,
         signature: DUMMY_ECDSA_SIG
       }),
     signUserOpHash: async (userOpHash: Hex) =>
       encodeSmartSessionSignature({
```

Comment:
add on to above comments, in module-sdk also encodeSmartSessionSignature uses single permissionId